homeassistant/components/sensor/youless.py

```diff
@@ -0,0 +1,119 @@
+"""
+@ Author      : Gerben Jongerius
+@ Date        : 04/29/2018
+@ Description : Youless Sensor - Monitor power consumption. This component will add the following sensors
+                   - Current power consumption (in W)
+                   - Current tick count, since the Youless meter started running (in W)
+"""
+import json
+import logging
+from datetime import timedelta
+from urllib.request import urlopen
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_VARIABLES
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+SENSOR_POWER = 'pwr'
+SENSOR_POWER_NET = 'net'
+SENSOR_P_HIGH = 'p2'
+SENSOR_P_LOW = 'p1'
+SENSOR_GAS = 'gas'
+
+DOMAIN = 'youless'
+CONF_HOST = "host"
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Optional(CONF_HOST): cv.string,
+        vol.Optional(CONF_MONITORED_VARIABLES,
+                     default=[SENSOR_POWER, SENSOR_POWER_NET]): vol.All(
+            cv.ensure_list, vol.Length(min=1),
+            [vol.In([SENSOR_POWER, SENSOR_POWER_NET, SENSOR_P_LOW,
+                     SENSOR_P_HIGH, SENSOR_GAS])])
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+SENSOR_PREFIX = 'youless_'
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_TYPES = {
+    SENSOR_POWER: ['Usage', 'usage', 'W', 'mdi:flash'],
+    SENSOR_POWER_NET: ['Power', 'meter', 'W', 'mdi:gauge'],
+    SENSOR_P_LOW: ['Power Low', 'meter_low', 'kWh', 'mdi:flash'],
+    SENSOR_P_HIGH: ['Power High', 'meter_high', 'kWh', 'mdi:flash'],
+    SENSOR_GAS: ['Gas', 'gas', 'm3', 'mdi:fire']
+}
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    host = config.get(CONF_HOST)
+    sensors = config.get(CONF_MONITORED_VARIABLES)
```

Comment:
Use [`CONF_MONITORED_CONDITIONS`](https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#0-common) instead of `CONF_MONITORED_VARIABLES`.

Reply:
I've changed it to the suggested CONF_MONITORED_CONDITIONS, will update the documentation for the sensor as well